src/SignIn/SignInScreen.js

```diff
@@ -26,6 +39,55 @@ const styles = StyleSheet.create({
 });
 
 export default function SignInScreen({ navigation }) {
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [number, setNumber] = useState('');
+  const [address, setAddress] = useState('');
+  const [hidePass, setHidePass] = useState(true);
+
+  const lastNameInput = useRef();
+  const emailInput = useRef();
+  const passwordInput = useRef();
+  const numberInput = useRef();
+  const addressInput = useRef();
+
+  const handleSignUp = () => {
+    base('Users').create([
+      {
+        fields: {
+          email,
+          password,
+          'first name': firstName,
+          'last name': lastName,
+          'phone number': number,
+          address,
+        },
+      },
+    ], (err, records) => {
+      if (err) {
+        console.error(err);
+        return;
+      }
+      records.forEach((record) => {
+        console.log(record.getId());
```

Comment:
Same as above, we don't want to keep console.log statements bc they can accumulate and be confusing for devs if you don't know where they originate and they don't give any error info to the user